src/main/java/com/rehab/model/Patient.java

```diff
@@ -0,0 +1,51 @@
+package com.rehab.model;
+
+public class Patient extends AbstractNamedEntity {
+    private int insuranceNumber;
+    private Doctor doctor;
+    private PatientState patientState;
+    private String diagnosis;
+
+    public Patient() {
```

Comment:
The same as with Cure class.

Reply:
Initialized patientState with TREATING in c04ab95